src/buildercore/cfngen.py

```diff
@@ -633,6 +634,11 @@ def _title_has_been_updated(title, section):
             if title_in_old['Type'] == 'AWS::EC2::Instance':
                 for property_name in EC2_NOT_UPDATABLE_PROPERTIES:
                     title_in_new['Properties'][property_name] = title_in_old['Properties'][property_name]
+
+        if 'Type' in title_in_old and title_in_old['Type'] == 'AWS::RDS::DBInstance':
+            for property_name in RDS_NOT_UPDATEABLE_PROPERTIES:
+                title_in_new['Properties'][property_name] = title_in_old['Properties'][property_name]
+
```

Comment:
remove changes to this file if satisfied. Rebasing and removing the commit would be the best way since there's no squash in the project

Reply:
change reverted. I don't mind git log noise.